pkg/source/openstack/client.go

```diff
@@ -772,6 +781,10 @@ func generateNetworkInfo(info map[string]interface{}) ([]networkInfo, error) {
 
 // SanitizeVirtualMachineImport is used to sanitize the VirtualMachineImport object.
 func (c *Client) SanitizeVirtualMachineImport(vm *migration.VirtualMachineImport) error {
+	if pointer.BoolDeref(vm.Spec.GracefulShutdown, false) {
```

Comment:
the pointer value is `false`, and the log message is `no need to activate the 'GracefulShutdown' option`

to run better, the openstack client can silently neglect the option, instead of report error strictly

Reply:
Done